pkg/agent/plugin/workloadattestor/docker/http.go

```diff
@@ -0,0 +1,31 @@
+package docker
+
+import (
+	"net/http"
+
+	"golang.org/x/time/rate"
```

Comment:
This is (IMO) un-testable. We should invest in an internal rate limiting implementation, or define an interface for `*rate.Limiter` and use that in spiffe/spire.